.cargo/config.toml

```diff
@@ -0,0 +1,2 @@
+[alias]
+w = "watch -x run"
```

Comment:
Please put this in `~/.cargo/config.toml` instead, I don't see the reason to share this alias with everyone else.